alerts/cluster-monitoring-operator/NodeClockNotSynchronising.md

```diff
@@ -0,0 +1,66 @@
+# NodeClockNotSynchronising
+
+## Meaning
+
+This alert signifies issues with the NTP server on a node (mentioned in the alert).
+When the affected node is a master, this specifically becomes an issue
```

Comment:
Does this mean masters should be severity of critical ? While workers severity of warning ?